### ISSUE TITLE
dota2 comends

### ISSUE THREAD

**Issue by skaavhai** [ValveSoftware/Dota2-Gameplay#12984]
### Cosmetic Item Name

commends

### Description

my commends does not count since past 2 months

### Example Match ID

7394647282

**Comment by ligaapl** [comment]
Yes. I have 2172 commends a month ago and commends still not counted after profile update! ID 113184848

**Comment by Aluerie** [comment]
d #11278
